threatx/info.json

```diff
@@ -0,0 +1,353 @@
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "ip_address",
+          "value": "",
+          "tooltip": "IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\".",
+          "description": "Specify IP address which you want to block in Threat X. IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\"."
+        }
+      ]
+    },
+    {
+      "operation": "blacklist_ip",
+      "title": "Blacklist IP Address",
+      "description": "Add IP Address entry to blacklist in Threat X.",
+      "category": "investigation",
+      "annotation": "blacklist_ip",
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "ip_address",
+          "value": "",
+          "tooltip": "IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\".",
+          "description": "Specify IP address which you want to blacklist in Threat X. IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\"."
+        },
+        {
+          "title": "Description",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "description",
+          "value": "",
+          "tooltip": "IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\".",
+          "description": "Specify the description of IP address in the blacklist."
+        }
+      ]
+    },
+    {
+      "operation": "unblacklist_ip",
+      "title": "Unblacklist IP Address",
+      "description": "Remove IP Address entry from blacklist in Threat X.",
+      "category": "investigation",
+      "annotation": "unblacklist_ip",
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "name": "ip_address",
+          "value": "",
+          "tooltip": "IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\".",
+          "description": "Specify IP address which you want to unblacklist in Threat X. IP address or CIDR, for example: \"10.1.1.1\" or \"10.1.1.0/24\"."
+        }
+      ]
+    },
+    {
+      "operation": "whitelist_ip",
+      "title": "Whitelist IP Address",
+      "description": "Add IP Address entry to whitelist in Threat X.",
+      "category": "investigation",
+      "annotation": "whitelist_ip",
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": false,
```

Comment:
This parameter is un-required? Can you please check.
Also, check for other parameters of this operation.